web/xfa_layer_builder.css

```diff
@@ -17,6 +17,37 @@
   --unfocused-field-background: url("data:image/svg+xml;charset=UTF-8,<svg width='1px' height='1px' xmlns='http://www.w3.org/2000/svg'><rect width='100%' height='100%' style='fill:rgba(0, 54, 255, 0.13);'/></svg>");
 }
 
+.xfaLayer .highlight {
```

Comment:
This attempts to mimic the current highlighting in text_layer_builder.css. Alternatively, I was think we could create a  .textHighlighter class and share the code, but it's currently small enough and different enough I'm not sure it's worth changing.

Reply:
That seems fine for now, and we can always change it in the future (if needed).